apps/modernization-ui/src/apps/page-builder/components/AddQuestion/fields/DataMartFields.tsx

```diff
@@ -14,8 +14,9 @@ type Props = {
     editing?: boolean;
     page?: number;
     questionId?: number;
+    disableWhenStandard?: boolean;
```

Comment:
Issue: Updated the story to hide the `DataMartFields` when editing a `standard` question. With that change, no modifications need to be made to the `DataMartFields` component